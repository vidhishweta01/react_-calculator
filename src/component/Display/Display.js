import propTypes from 'prop-types';

const Display = ({ total }) => (
  <h1>
    {' '}
    {total}
    {' '}
  </h1>
);

Display.propTypes = {
  total: propTypes.string,
};

Display.defaultProps = {
  total: '0',
};

export default Display;
