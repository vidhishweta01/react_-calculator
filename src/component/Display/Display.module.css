.h1 {
  font-size: xx-large;
  height: 60px;
  text-align: right;
  padding-right: 1vw;
  border: 2px solid #d1d1d1;
  background-color: rgb(131, 134, 134);
  margin-bottom: 0;
  color: white;
  padding-top: 1vh;
}
