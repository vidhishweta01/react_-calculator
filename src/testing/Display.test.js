import React from 'react';
import Display from '../component/Display';
import renderer from 'react-test-renderer';

describe('Display component', () => {
    it('matches the snapshot', () => {
      const display = renderer.create(<Display />).toJSON();
      expect(display).toMatchSnapshot();
    });
  });