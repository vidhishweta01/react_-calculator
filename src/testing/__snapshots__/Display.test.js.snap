// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Display component matches the snapshot 1`] = `
<h1
  className="h1"
>
  0
</h1>
`;
